.husky/scripts/check-testing.sh

```
@@ -0,0 +1,33 @@
+    # Check if the tests passed 
+    if [ $exit_code -ne 0 ]; then
+        echo "❌ Test execution failed $exit_code"
+        exit $exit_code
+    fi
+
+    echo "✅ All tests passed"
+
+    echo "$test_output" | awk '/Test result:/ {
+        sub(/OK/, "'${YELLOW}'OK'${NC}'");
+        print
+    '}
```

Shellcheck (0.10.0)

[warning] This } is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)